### ISSUE TITLE
Anirip not downloading subtitles

### ISSUE THREAD

**Issue by Dokman** [s32x/anirip#25]
So when you use the -l or --language or edit them in main.go does download it at english too...

**Comment by s32x** [comment]
What is the command you are using?? I need the show name and language you are trying to download. Come on man, if you want help you need to provide me with more context.

**Comment by Dokman** [comment]
anirip.exe user pass --language esp linktoserie

**Comment by Dokman** [comment]
oh yes and i recompiled with the newest sources of the repo so... idk why is not working...

**Comment by davidchalifoux** [comment]
@Dokman He asked for the series list and you just gave him "linktoserie". He can't debug anything without knowing what series, as the program works for everything else.

**Comment by Dokman** [comment]
any anime doesn't work i tried a lot of animes with spanish subtitles but the program doesn't put it in the video...

**Comment by davidchalifoux** [comment]
You are right, it will only download english subtitles. This is definitely a bug.

**Comment by Dokman** [comment]
so it's a bug during a lot of moths because i tried a lot of time before i opened a isssue

**Comment by Dokman** [comment]
@davidchalifoux it has been passed a long time... it won't be fixed xD

**Comment by davidchalifoux** [comment]
@Dokman I might work on a fix for it sometime this week. You could also give it a go.

I'm considering making a user-friendly fork.

**Comment by s32x** [comment]
Feel free to make a PR as well or I can just merge the fork when you make it public. Sorry I don’t get around to this project very much.

**Comment by s32x** [comment]
Also, I can’t even begin to debug the issue without the answer to the question I asked at the beginning of the issue xD
  
  
  
>   
>
>

**Comment by s32x** [comment]
Please provide a command

**Comment by Dokman** [comment]
WTF use anirip -user myuser -pass mypass -l https://www.crunchyroll.com/es-es/the-rising-of-the-shield-hero/episode-20-battle-of-good-and-evil-783057

**Comment by s32x** [comment]
Is that a series URL or an episode URL?

**Comment by Dokman** [comment]
anirip -user myuser -pass mypass -l esp https://www.crunchyroll.com/es-es/the-rising-of-the-shield-hero/

**Comment by s32x** [comment]
Very good. If I happen to find the time I'll be more than happy to fix this issue.

**Comment by AlvoErrado2** [comment]
Did you manage to find out something about this error?

**Comment by turtletowerz** [comment]
The subtitle downloading was updated in late 2019 and this thread was early 2019. If it still doesn't work for specific languages please create a new issue for it, his one can probably be closed.
